lib/content-services/content-node-selector/content-node-dialog.service.ts

```diff
@@ -189,7 +189,11 @@ export class ContentNodeDialogService {
     }
 
     private hasEntityCreatePermission(entry: MinimalNodeEntryEntity): boolean {
-        return this.contentService.hasPermission(entry, 'create');
+        return this.contentService.hasPermission(entry, 'create') && !this.isSite(entry);
```

Comment:
this check shoudl not be in and here otherwise the name of the method is wrong

Reply:
so it will be mos tporbably in and here 
```
            const data: ContentNodeSelectorComponentData = {
                title: title,
                actionName: action,
                currentFolderId: contentEntry.parentId,
                imageResolver: this.imageResolver.bind(this),
                rowFilter : this.rowFilter.bind(this, contentEntry.id),
                isSelectionValid: this.hasEntityCreatePermission.bind(this) && !this.isSite(entry),
                select: select
            };
```

Reply:
I have made the suggested changes, please check. Thank you.